app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="<%= dom_id group %>">
+<div class="group-display">
+<div class="group-list">
+  <div class="img"><img src="<%= group.icon %>"> </div>
+  <div class="group-name"><%= link_to group.name, user_group_expenditures_path(group_id: group.id) %> </div>
+  <span><%= number_to_currency(@expenditures.sum(&:amount)) %> </span>
+  </div>
+  <span class="date"><%= group.created_at.to_date %></span>
```

Comment:
- [ ] In the group index the amount we are showing for each group is the same. Please fix the logic only to get the sum for each group separately.

As you can see here, we have the same amount for both groups, but we don't have any transactions for sports but still we are seeing $68. Please Fix this.

![Screenshot from 2022-09-23 00-45-28](https://user-images.githubusercontent.com/26767383/191836400-219ca0f9-4e66-4e3c-95d1-94641d171fb9.png)